### DISCUSSION TITLE
Audi MHI2Q connect and traffic

### DISCUSSION THREAD

**Discussion by georgi071** [Mr-MIBonk/M.I.B._More-Incredible-Bash#94] [Q&A]
Hello.

Has anyone succeeded to enable the AUDI connect and / or traffic service after pathing with MIB tool? 
My version is MHI2Q_ER_AUG22_P5092 and MU software 1329

thanks

**Comment by harman-f** [comment]
No, M.I.B can not activate this. So far a server is needed to do this.
If your unit is stock, get this function from Audi.

In case of retrofit, there are custom servers which offer this. Of course, for a monthly fee, very similar in price to what Audi is charging...

**Reply by georgi071** [reply]
thank you

**Reply by georgi071** [reply]
I get some input that you could adjust a FeC key and maybe have it again as i used to a couple of years ago.

**Reply by TerryUK** [reply]
I'm wondering whether there isn't a missing FeC in the app that's removed the screen. I reckon two things are going on here is the FeC for the screen and the enablement. I say this since the Seat one omitted the Seat apps so without this you don't get the app screen. So I'm thinking it's been tidied out even thou it's been previously enabled ?.

**Reply by georgi071** [reply]
> I'm wondering whether there isn't a missing FeC in the app that's removed the screen. I reckon two things are going on here is the FeC for the screen and the enablement. I say this since the Seat one omitted the Seat apps so without this you don't get the app screen. So I'm thinking it's been tidied out even thou it's been previously enabled ?.

I can see the AUDI connect menu in my car, but it says it has expired. Once inside I can only go to software updates and a couple of other options but no traffic and of course no remote lock/unlock, GPS location from my phone, etc.  I just need this live traffic, the rest are not that important to me.

Another strange thing is that through the "my.Audi" site when i log in to my account and car, when i try to see if I can buy through the "Functions on Demand menu" a new activation, the option is not available. That was also the case even before patching with MIB tool.

**Reply by TerryUK** [reply]
Well as I said as long as you have all your original FeCs now showing plus the long life navigation FeC there shouldn't be a problem. The Seat one had a couple or so of FeCs trimmed back where it was evident that the developer that did the addfec.txt template thought you didn't now need them now. So that's what I was saying just check you haven't lost anything which has upset what you had working before.

**Reply by georgi071** [reply]
Could it be another FeC alteration that i missed somehow or the live server communication and authentication must take place?

**Reply by Lanye-z** [reply]
> No, M.I.B can not activate this. So far a server is needed to do this. If your unit is stock, get this function from Audi.
> 
> In case of retrofit, there are custom servers which offer this. Of course, for a monthly fee, very similar in price to what Audi is charging...

Hello, do you have any tutorials on how to build this service on personal server? Maybe we could build this service with a personal home server to suit our own needs?

**Comment by mjfrank** [comment]
Is anyone able to share additional information the modifications needed on the MIB2 and the custom server software/scripts required for this?

**Reply by harman-f** [reply]
There are custom build servers running. But as far as I have seen they are all payed…

Sadly, I have not seen a community solution so far.
